.github/workflows/maven-cd.yml

```diff
@@ -0,0 +1,65 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+
+name: cd
+on:
+  workflow_call:
+    secrets:
+      MAVEN_USERNAME:
+        required: true
+        description: Maven username used for deploying the plugin jar to Jenkins Artifactory Repository
+      MAVEN_TOKEN:
+        required: true
+        description: Maven token used for deploying the plugin jar to Jenkins Artifactory Repository
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    outputs:
+      should_release: ${{ steps.verify-ci-status.outputs.result == 'success' && steps.interesting-categories.outputs.interesting == 'true' }}
+    steps:
+      - name: Verify CI status
+        uses: jenkins-infra/verify-ci-status-action@v1.2.0
+        id: verify-ci-status
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          output_result: true
+
+      - name: Release Drafter
+        uses: release-drafter/release-drafter@v5
```

Comment:
I guess we could actually pin this to a specific version in the shared workflow: we would get DB bumps, which PRs would not test, but after merging we would be sure to run `jenkins-infra-test-plugin` again to make sure everything still works.

Reply:
are we talking `v5.x.x` level of pinning or commit sha pinning? 😅

Reply:
I meant `v5.x.x`, I am not _that_ paranoid!

Reply:
(I did see a tool recently that let you pin actions to hashes and still get them updated.)